TerminalDocs/customize-settings/profile-settings.md

```diff
@@ -248,6 +263,27 @@ This sets the percentage height of the cursor starting from the bottom. This wil
 
 ___
 
+## Keyboard settings
+
+### AltGr aliasing ([Preview](https://aka.ms/terminal-preview/))
+
+This allows you to control if Windows Terminal will treat <kbd>ctrl+alt</kbd> as an alias for <kbd>AltGr</kbd>.
```

Comment:
Capitalize keystrings consistently, at least in the same sentence

```suggestion
This allows you to control if Windows Terminal will treat <kbd>Ctrl+Alt</kbd> as an alias for <kbd>AltGr</kbd>.
```

Reply:
It looks like other parts of this document use lowercase . . . so feel free to do what is best